collections-csharp/MergingDictionariesInCSharp/MergingDictionariesInCSharp/Methods/ConcatMethod.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace MergingDictionariesInCSharp
+{
+    public partial class Methods
+    {
+        public static Dictionary<TKey, TValue> ConcatMethod<TKey, TValue>(params Dictionary<TKey, TValue>[] dictionaries)
```

Comment:
We should be sure there is also a test showing the expected ArugmentException when the two dictionaries contain the same key. This is explicitly called out in the article that the dictionaries must contain unique keys, so it would be good to have a test confirming that as well.

Reply:
Added Exception test